.gitignore

```diff
@@ -79,4 +79,5 @@ jspm_packages/
 .pnp.*
 
 .idea
-.vscode
\ No newline at end of file
+.vscode
+legacy-app
```

Comment:
`legacy-app/` is added to .gitignore so I'm not sure if you meant to publish an example legacy app or not? asking just in case this should be in the PR.

Reply:
ahh, added this for testing but removed it. Do you think it's a good idea to bundle this in the repo or is that too confusing? Perhaps introduce an "additional examples" folder for less common cases like this?